### PR TITLE
Containers: Increase zypper timeout and fix script_run logic

### PR DESCRIPTION
**Pull request by pdostal**
- Errors: [docker container exec zypper in](https://openqa.suse.de/tests/7531053#), [docker container commit](https://openqa.suse.de/tests/7587608#step/docker/206)
- Related ticket: [poo#101484](https://progress.opensuse.org/issues/101484)
- Verification run: [1](http://pdostal-server.suse.cz/tests/12646)

### CODE REVIEW

File: lib/containers/utils.pm [hunk 1]
```diff
@@ -155,12 +155,14 @@ sub basic_container_tests {
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
     # Containers' state can be saved to a docker image
-    if (script_run("$runtime container exec $container_name zypper -n in curl", 300)) {
+    my $ret = script_run("$runtime container exec $container_name zypper -n in curl", 200);
```
Comment (jlausuch) [inline lib/containers/utils.pm:158]: why decreasing the timeout here?

Comment (jlausuch) [reply lib/containers/utils.pm:158]: Should we use script_retry here? Similar to  https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13542

Comment (pdostal) [reply lib/containers/utils.pm]: Sorry, I increased it to 420 now.

I don't think we need `script_retry()`.

**Review by b10n1k — APPROVED** [review]
LGTM

File: lib/containers/utils.pm [hunk 2]
```diff
@@ -155,7 +155,9 @@ sub basic_container_tests {
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
     # Containers' state can be saved to a docker image
-    if (script_run("$runtime container exec $container_name zypper -n in curl", 300)) {
+    my $ret = script_run("$runtime container exec $container_name zypper -n in curl", 420);
```
Comment (dzedro) [inline lib/containers/utils.pm:158]: I would not be so humble with timeout increase.
Sometimes it's better to have longer timeout, it has time to return reason of the error.
But you know better, great If 420 is good enough. :+1:

Comment (pdostal) [reply lib/containers/utils.pm]: You're right, I did `600`.

**Review by jlausuch — APPROVED** [review]

**Pull request merged**